### ISSUE TITLE
F-Droid inclusion

### ISSUE THREAD

**Issue by 532910** [m3sv/PlainUPnP#1]
F-Droid inclusion issue: https://gitlab.com/fdroid/rfp/issues/903

**Comment by IzzySoft** [comment]
@m3sv if you could at least tag your releases and attach the APK to the releases, I could put your app into my repo until you're ready for the official one. Just ping me if you have that, and I'd go for it :smile:

**Comment by Shahav** [comment]
So sorry,  yes...Of course you can.
Sherry

**Comment by IzzySoft** [comment]
Thanks for the APK! Added it to my repo (should show up in about 11h with the next sync). You still need to add a license to your repo (as the project you forked from uses GPL-3.0, this will be what you must use I guess, so I've set that already; of course you might decide for "GPL-3.0-or-later" (while I just put "GPL-3.0-only" for now).

Then, establishing a [Fastlane file structure](https://gitlab.com/snippets/1895688) to provide screenshots (and to maintain the app description) would be a good idea. Not mandatory, but highly recommended.

Thanks again, and best luck!

**Comment by Pentaphon** [comment]
@m3sv Thanks for working on this. Your app would be great for F-Droid.

**Comment by m3sv** [comment]
@IzzySoft Thank you very much for all the help! Sorry for taking soo long to reply. Will try to implement everything ASAP. Thank you again! 😀

**Comment by IzzySoft** [comment]
@m3sv all fine – I know how it feels with "time needed". Just let me know when the license is decided (so I can update that in my repo), and when something else is I should pick (like screenshots from fastlane – or even the app being ready for the RFP to proceed).

**Comment by Pentaphon** [comment]
@m3sv Any progress on adding this promising app to F-Droid? I think adding it will boost your userbase significantly since FOSS users are very welcoming of new, useful apps.

**Comment by m3sv** [comment]
Right now I'm about to release 2.1 into beta, it should be the most feature complete and stable release so far (at least I hope so, lol), if this goes well, then the next stop is definitely FDroid. Stay tuned.

**Comment by m3sv** [comment]
Fixed... I guess? 

@Pentaphon 
@IzzySoft 
@licaon-kter 
@532910
Thanks for helping out!

**Comment by m3sv** [comment]
Fixed... I guess?

@Pentaphon
@IzzySoft
@licaon-kter
@532910
Big kudos for helping out!

**Comment by licaon-kter** [comment]
Yes and no, suddenly versionCode and versionName went away so [autoupdate ckeck](https://gitlab.com/fdroid/fdroiddata/-/commit/8b0a0516cf60506ca651fd56e32661215a31ea33#49c7ee91c3ea9ede3281feacce9d934bd6881e49_40_40) will fail to find any new versions.

Any chance to bring them back?

**Comment by m3sv** [comment]
Ouch, that's unfortunate. I'll return buildSrc back this evening.

Thanks for letting me know. 🍻

**Comment by licaon-kter** [comment]
Well, it's not about buildSrc perse, but about a place where we can grep some static info about the versionCode/Name.

**Comment by m3sv** [comment]
Yeah, right now they're constructed from multiple sources, AFAIK.

So the requirement is that they should be static, correct?

**Comment by licaon-kter** [comment]
We don't run (yet) any dynamic functions in the checkupdate stage so static is needed.

**Comment by m3sv** [comment]
@licaon-kter Pushed the change. Hope the format is okay.
